elements/rh-secondary-nav/demo/demo.css

```diff
@@ -1,12 +1,12 @@
 :root {
-  --pfe-theme--font-family: "Red Hat Text", "Overpass", Helvetica, Arial, sans-serif;
-  --pfe-theme--font-family--heading: "Red Hat Display", "Overpass", Helvetica, Arial, sans-serif;
+  --pfe-theme--font-family: RedHatText, "Red Hat Text", "Noto Sans Hebrew", "Noto Sans Japanese", "Noto Sans Korean", "Noto Sans Traditional Chinese", "Noto Sans Simplified Chinese", "Noto Sans Arabic", Overpass, Helvetica, Arial, sans-serif;
```

Comment:
ideally, we'd convert rem-to-px in tokens, merge that, then release, and then double-check all these token values.

I'm approving the PR bc LGTM, but leaving this comment to block pending your discretion whether or not to do the above (which really shouldn't take too long :crossed_fingers: )